theseus/labs/lie/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from .types import ltype, SE3, SO3
+from .lie_tensor import LieTensor, new
```

Comment:
Should we also pull functional API here so they are accessible at the `lie.` level too.